db/migrate/20170207212720_alter_photos_add_image.rb

```diff
@@ -0,0 +1,5 @@
+class AlterPhotosAddImage < ActiveRecord::Migration[5.0]
+  def change
+    add_column :photos, :image, :string
```

Comment:
Your `photos` table didn't have a field to connect to carrierwave.  I added the `image` field here (you were going in the direction of adding a `photo`, which could work, but it would be a little more confusing what a `photo` is).